packages/api/src/server.ts

```diff
@@ -12,25 +12,51 @@ import {
 	getAmplifyServerContext,
 } from '@aws-amplify/core/internals/adapter-core';
 
-import { __amplify, V6Client } from '@aws-amplify/api-graphql';
+import { __amplify, V6Client, V6ClientSSR } from '@aws-amplify/api-graphql';
 
 export type {
 	GraphQLResult,
 	GraphQLReturnType,
 } from '@aws-amplify/api-graphql';
 
+// move this import and ServerClientGenerationParams into `api-graphql`
```

Comment:
Not a blocker, but if you are updating this PR already it would be worth adding this so that we don't lose track of existing TODOs:

```suggestion
// TODO: move this import and ServerClientGenerationParams into `api-graphql`
```